SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -92,6 +92,25 @@ void FSpatialGDKEditorToolbarModule::StartupModule()
 		});
 	}
 
+	FEditorDelegates::PostPIEStarted.AddLambda([this](bool bIsSimulatingInEditor)
+	{
+		if (GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking() && GIsAutomationTesting)
```

Comment:
Nit: I would very very slightly pref GIsAutomationTesting condition first to GetDefault<UGeneralProjectSettings>()->UsesSpatialNetworking().

Don't have to do this. It's insignificant.